### PR TITLE
Add htmlzip download

### PR DESCRIPTION
**Pull request by sciencewhiz**
This allows the WPILib installer to download the API docs like we do with Java and C++
Credit for rtd_patch.py goes to @TheTripleV from https://github.com/wpilibsuite/frc-docs/blob/main/source/_extensions/rtd_patch.py Adapted to remove the photofinish part

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -21,6 +21,10 @@
     "robotpy_sphinx.all",
 ]
 
+local_extensions = [
+    "_extensions.rtd_patch",
+]
+
```
Comment (TheTripleV) [inline docs/conf.py:27]:
```suggestion
extensions += local_extensions
```

**Comment by virtuald** [comment]
I think this would be better suited for https://github.com/robotpy/robotpy-sphinx-plugin/blob/main/robotpy_sphinx/all.py, then we get it for free in all RobotPy projects

**Comment by sciencewhiz** [comment]
I opened https://github.com/robotpy/robotpy-sphinx-plugin/pull/7 so now this only enables htmlzip in RTD

**Comment by virtuald** [comment]
I found the issue that y'all originally commented on at RTD (https://github.com/readthedocs/readthedocs.org/issues/3242), and they posted this solution instead?

https://github.com/readthedocs/test-builds/blob/f6000f53380887463560499d82e5d0e7bfad6150/.readthedocs.yaml#L9

**Comment by virtuald** [comment]
Trying that in f5ed2b2401600ff3de9ddb91726a695b0e6c65eb

**Pull request closed without merging**